library/solr

```diff
@@ -8,17 +8,22 @@ Maintainers: The Apache Solr Project <dev@solr.apache.org> (@asfbot),
 GitRepo: https://github.com/apache/solr-docker.git
 GitFetch: refs/heads/main
 
-Tags: 9.0.0, 9.0, 9, latest
+Tags: 9.1.0, 9.1, 9, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 46493704912ffeb3e70d2d9fb91701089e65b8e1
+GitCommit: c07ee789c60dd4be20d63e8172180ac8a2bd919e
+Directory: 9.1
+
+Tags: 9.0.0, 9.0
```

Comment:
Just a clarification, are you keeping `9.0.0` around since there could be a `9.0.1`? In other words, is it still a "supported" version of solr?

Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Hmm, I think only 9.1.0 should be supported in the 9.x series. Will check..

Reply:
Is there an ETA on the 9.1.0 docker image release?

Reply:
+1 for merging this as-is, keeping 9.0.0 "supported" for now. Then we can "clean up" as a follow-up task.

Reply:
Similarly, y'all should consider whether 8.11.x is going to have any more updates in the future (because the same comment applies).

Reply:
8.11 is 100% going to have updates, likely one in a month or two.